resources/lang/en/items.php

```diff
@@ -12,6 +12,8 @@
         'price'     => 'Price',
         'size'      => 'Size',
         'type'      => 'Type',
+        'item_id'   => 'Parent Item ID',
```

Comment:
Our convention is just "Parent Item" for the parent field, "ID" is a too technical for our users :)